app/assets/javascripts/react_on_rails.js

```diff
@@ -150,4 +150,19 @@
 
     return consoleReplay;
   }
+
+  ReactOnRails.provideClientReact = function() {
```

Comment:
Should we expose this globally?

I suggest we do it the same we declared:

https://github.com/rstudner/react_on_rails/blob/react-0.14-support/app/assets/javascripts/react_on_rails.js#L78
```javascript
  function createReactElement(componentName, propsVarName, props, domId, trace, generatorFunction) {
```